internal/storage/postgre.go

```diff
@@ -23,32 +25,181 @@ func NewPostgreBase(db *pgx.Conn, log *zap.Logger) *Postgre {
 	}
 }
 
+func CheckDB(db *pgx.Conn) error {
+	_, err := db.Exec(context.Background(), "CREATE TABLE IF NOT EXISTS urls (id TEXT PRIMARY KEY, url TEXT)")
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
 func (p *Postgre) Ping() error {
 	p.Log.Debug("Проверяем статус соединения с БД")
 	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
 	defer cancel()
 	if err := p.db.Ping(ctx); err != nil {
-		p.Log.Debug("Ошибка соединения с БД")
+		p.Log.Error("Ошибка соединения с БД")
 		fmt.Println(err)
 		return err
 	}
-	p.Log.Debug("Есть соединение с БД")
+	p.Log.Info("Есть соединение с БД")
 	return nil
 }
 
 func (p *Postgre) AddURL(data URLData) error {
-	//TODO implement me
-	panic("implement me1")
+	p.Log.Debug("Добавляем URL в базу данных", zap.String("url", data.URL))
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
+	defer cancel()
+	_, err := p.db.Exec(ctx, "INSERT INTO urls (id, url) VALUES ($1, $2)", data.ID, data.URL)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (p *Postgre) AddBatchURL(urls map[string]ReqBatchJSON) error {
```

Comment:
CHANGE REQUEST: you should provide `context`

Reply:
done